src/edit_mode/vi/command.rs

```diff
@@ -113,6 +113,10 @@ where
             let _ = input.next();
             Some(Command::RepeatLastAction)
         }
+        Some('o') => {
+            let _ = input.next();
+            Some(Command::SwapCursorAndAnchor)
+        }
```

Comment:
I think in theory this dispatch from the parser is in sufficient as it would compete with the normal mode `o` (open insert mode in the line below) and we don't discriminate between the modes here.

But as we don't have that `o` implemented yet (as far as i can tell) this should be fine.